R/pkg/R/functions.R

```diff
@@ -3035,6 +3036,19 @@ setMethod("array_position",
             column(jc)
           })
 
+#' @details
+#' \code{flatten}: Transforms an array of arrays into a single array.
```

Comment:
not a big deal but let's match the doc too with Python and Scala (not to SQL one) since that's usual so far. Please fix it in the PRs for other functions later.